recipes/thermohubclient/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: thermohubclient
+  version: {{ version }}
+
+source:
+  url: https://github.com/thermohub/thermohubclient/archive/v{{ version }}.tar.gz
+  sha256: 8a6901e34fcd87e88c098134750807f8cbf95080ce45f078bf2ff48955cf5f15
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja  # [win]
+    - make  # [unix]
+  host:
+    - pybind11
+    - nlohmann_json
+    - numpy
+    - python
+    - jsonarango
+    - velocypack
+    - libcurl
+  run:
+    - python
+test:
+  imports:
+    - thermohubclient
+  commands:
+    - test -d ${PREFIX}/include/thermohubclient  # [unix and osx]
```

I think this test should be
```suggestion
    - test -d ${PREFIX}/include/ThermoHubClient  # [unix]
    - test -f ${PREFIX}/lib/libThermoHubClient${SHLIB_EXT}  # [unix]
    - test -f ${PREFIX}/include/ThermoHubClient/ThermoHubClient.h  # [unix]
```

---

Linux is a bit more strict regarding upper and lower case letters